################################################################################
SHORT DESCRIPTION: 
################################################################################
gitignore: Add missing generated files.

################################################################################
LONG DESCRIPTION: 
################################################################################
gitignore: Add missing generated files.

################################################################################
CHANGELOG 
################################################################################
Author(s):
Eric Chanudet <chanudete@ainfosec.com>

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
Yes, should be upstreamed.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
--- a/.gitignore
+++ b/.gitignore
@@ -15,6 +15,8 @@
 cscope.*
 TAGS
 VERSION
+EXTRAVERSION
+*linux-libtool
 
 # ac/am
 Makefile.in
@@ -30,6 +32,7 @@ Makefile.in
 /m4/
 /missing
 /stamp-h1
+/mk/config.log
 
 # executables
 /drivers/lock-util
@@ -53,6 +56,7 @@ Makefile.in
 /vhd/lib/test/test-snapshot
 /tapback/tapback
 /test-driver
+/cpumond/cpumond
 
 # generated makefiles
 /Makefile
@@ -74,7 +78,8 @@ Makefile.in
 /vhd/lib/Makefile
 /vhd/lib/test/Makefile
 /tapback/Makefile
-mk/config.log
+/cpumond/Makefile
+/drivers/crypto/Makefile
 
 # dist output
 /blktap-*.tar.*
